app/boards/shields/cradio/cradio.keymap

```diff
@@ -8,95 +8,87 @@
 #include <dt-bindings/zmk/keys.h>
 #include <dt-bindings/zmk/bt.h>
 
-
 &mt {
-  //  flavor = "tap-preferred";
-   // tapping_term_ms = <200>;
+//    flavor = "tap-preferred";
+//    tapping_term_ms = <200>;
 };
 
-/ { 
-
+/ {
     combos {
         compatible = "zmk,combos";
         combo_esc {
             timeout-ms = <50>;
             key-positions = <0 1>;
             bindings = <&kp ESC>;
         };
-        
-         combo_tab {
+
+        combo_tab {
             timeout-ms = <50>;
             key-positions = <10 11>;
             bindings = <&kp TAB>;
         };
-        
-            combo_ralt {
+
+        combo_ralt {
             timeout-ms = <50>;
             key-positions = <17 16>;
             bindings = <&kp RALT>;
         };
-        
-                    combo_lalt {
+
+        combo_lalt {
             timeout-ms = <50>;
             key-positions = <11 12>;
             bindings = <&kp LALT>;
         };
-        
-                           combo_lgui {
+
+        combo_lgui {
             timeout-ms = <50>;
             key-positions = <12 13>;
             bindings = <&kp LGUI>;
         };
-        
-        
-           combo_rgui {
+
+        combo_rgui {
             timeout-ms = <50>;
             key-positions = <17 18>;
             bindings = <&kp RGUI>;
         };
-        
-
-        
     };
 
-        keymap {
-                compatible = "zmk,keymap";
-                
-       		default_layer {
-		bindings = <
-		&kp Q &kp W &kp E &kp R &kp T 						&kp Y &kp U  &kp I    &kp O   &kp P 
-		&kp A &kp S &kp D &kp F &kp G 			        	        &kp H &kp J &kp K &kp L &kp QUOT
-		&mt LSFT Z &kp X &kp C &kp V &kp B				        &kp N &kp M  &kp CMMA &kp DOT &mt LSFT RET
-          				&mo 1 &kp LCTL  				&kp SPC &mo 2 
-		>;
-		};
+    keymap {
+        compatible = "zmk,keymap";
+        default_layer {
+            bindings = <
+            &kp Q       &kp W  &kp E  &kp R  &kp T         &kp Y    &kp U  &kp I     &kp O    &kp P
+            &kp A       &kp S  &kp D. &kp F  &kp G         &kp H    &kp J  &kp K     &kp L    &kp QUOT
+            &mt LSFT Z  &kp X  &kp C  &kp V  &kp B         &kp N    &kp M  &kp CMMA  &kp DOT  &mt LSFT RET
+                                      &mo 1  &kp LCTL      &kp SPC  &mo 2
+            >;
+        };
+
+        left_layer {
+            bindings = <
+            &kp NUM_1  &kp NUM_2  &kp NUM_3        &kp NUM_4        &kp NUM_5       &kp NUM_6  &kp NUM_7  &kp NUM_8  &kp NUM_9 &kp NUM_0
```

Comment:
I believe these number keycodes are deprecated; you can run the layers through https://zmk.dev/docs/codes/keymap-upgrader to quickly update them.

Reply:
Thanks for the pointer.